### ISSUE TITLE
Add Chromecast button/access from outside of show selection

### ISSUE THREAD

**Issue by abuettner93** [fredrikburmester/streamyfin#46]
**Describe the solution you'd like**
The Chromecast remote controls aren't available anywhere but on the page you start playing from. It would be nice to have them available from the main app menu, or generally anywhere (maybe a similar player control to that of playing media directly on the phone). 

**Additional context**
Player control on the media page:
![image](https://github.com/user-attachments/assets/73fa9a77-1887-4ce7-a1e6-09ec88b21503)

Lack of player control once streaming (away from media page):
![image](https://github.com/user-attachments/assets/a849467b-5981-498e-8f94-610f31624ac8)

**Comment by fredrikburmester** [comment]
Yeah this sounds like a good idea! The chromecast should be more "global".

**Comment by fredrikburmester** [comment]
Where do you think would be a better placement? @abuettner93

**Comment by fredrikburmester** [comment]
Improved in 0.4.2. Closing for now. Please reopen if the implementation is not good enough.
